lineage/Analyze.py

```diff
@@ -169,7 +266,7 @@ def get_stationary_distribution(transition_matrix):
     c = np.zeros((K + 1, 1))  # 9
     c[K, 0] = 1
     p = np.linalg.solve(BT_B, np.matmul(B.T, c)).T  # 11
-    assert np.allclose(np.matmul(transition_matrix.T, p.T), p.T)  # 12
+    # assert np.allclose(np.matmul(transition_matrix.T, p.T), p.T)  # 12
```

Comment:
Why comment this out?

Reply:
I was receiving some error about how `np.matmul` didn't like the transition matrix because it was nonsingular and I lazily commented it out.